.markdownlint.yml

```diff
@@ -18,6 +18,6 @@
 "MD049":
   "style": "asterisk"
 "MD013":
-  "line_length": 80
+  "line_length": 170
```

Comment:
I think @MalteHerrmann wanted to limit it to 80?

Reply:
@austinchandra had suggested 80, I didn't have any objections to it - 120 would be fine for me as well, 170 seems a bit long and also random IMO, 80 and 120 are widely adopted limits I think?